### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by 1-byte-man**
responding to #628  I added the database types to the API.md file

### CODE REVIEW

**Comment by aphelionz** [comment]
Thanks a lot @1-byte-man! @RichardLitt Can you give these a once over for grammar / etc? Otherwise, looks good to me.

**Comment by 1-byte-man** [comment]
>Thanks a lot @1-byte-man! @RichardLitt Can you give these a once over for grammar / etc? Otherwise, looks good to me.

ok there isn't problem. Sorry for the grammar I'm Venezuelan I'm gonna see what can I do to get it better

**Comment by aphelionz** [comment]
Estamos felices por sus contribuciones :)

File: API.md [hunk 1]
```diff
@@ -95,6 +95,15 @@ const db = await orbitdb.keyvalue('profile')
 
 ## Public Instance Methods
 
+Before to start you've to know that OrbitDB has different types of databases. Each one made to satisfy a different purpose. The databases that you can create are:
```
Comment (RichardLitt) [inline API.md]: Before starting, you should know that OrbitDB has different types of databases. Each one satisfies a different purpose. The databases that you can create are:

File: API.md [hunk 2]
```diff
@@ -95,6 +95,15 @@ const db = await orbitdb.keyvalue('profile')
...
+
+* [log](https://github.com/orbitdb/orbit-db/blob/master/API.md#orbitdblognameaddress): an imutable (write only) log database. Useful for transactions lists.
+* [feed](https://github.com/orbitdb/orbit-db/blob/master/API.md#orbitdbfeednameaddress): a mutable log database. Useful for comments of a blog.
```
Comment (RichardLitt) [inline API.md]: ...useful for blog comments.

File: API.md [hunk 3]
```diff
@@ -95,6 +95,15 @@ const db = await orbitdb.keyvalue('profile')
...
+* [keyvalue](https://github.com/orbitdb/orbit-db/blob/master/API.md#orbitdbkeyvaluenameaddress): Useful for load data from keywords or an id.
```
Comment (RichardLitt) [inline API.md]: Useful for loading data from keywords or an id.

File: API.md [hunk 4]
```diff
@@ -95,6 +95,15 @@ const db = await orbitdb.keyvalue('profile')
...
+* [docs](https://github.com/orbitdb/orbit-db/blob/master/API.md#orbitdbdocsnameaddress-options): a JSON documents database. Useful for user data or other structured data.
+* [counter](https://github.com/orbitdb/orbit-db/blob/master/API.md#orbitdbcounternameaddress): Useful for ordered data as could be an order list or a playlist.
```
Comment (RichardLitt) [inline API.md]: Useful for ordered data (like an ordered list or playlist).

**Comment by 1-byte-man** [comment]
> 
> 
> Estamos felices por sus contribuciones :)

thanks for speak me on spanish but I'll try to speak in english and do it better

File: API.md (hunk 1 above)
Comment (1-byte-man) [reply API.md]: > 
> 
> Before starting, you should know that OrbitDB has different types of databases. Each one satisfies a different purpose. The databases that you can create are:

ok thanks for the feedback I'll commit the changes

**Comment by RichardLitt** [comment]
¡Muchas Gracias! This is great. We really appreciate the work (and, really, no worries about English. We'd love more Spanish docs eventually.)

**Pull request merged**